manuscripts.json

```diff
@@ -97,6 +97,32 @@
       },
       "msas": ["Neuroscience"],
       "pdfUrl": "https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/10.1101/2022.11.08.515698/2022.11.08.515698.pdf"
+    },
+    "10.1101/2022.10.28.514241": {
+      "preprintDoi": "10.1101/2022.10.28.514241",
+      "status": {
+        "articleType": "Reviewed Preprint",
+        "status": "Published from the original preprint after peer review and assessment by eLife.",
+        "timeline": [
+          { "name": "Reviewed Preprint posted", "date": "2023-01-02" },
```

Comment:
I think this should be `"date": "2023-02-02"` if we're aiming to publish this today.

Reply:
Updated!